packages/react/src/components/Slug/slug-story.scss

```diff
@@ -164,3 +165,24 @@
 .sbdocs-slug .cds--toggletip-content {
   font-size: 0.875rem;
 }
+
+.slug-selectable-tile-container .cds--tile-group > div {
+  display: flex;
+}
+
+.slug-selectable-tile-container .cds--tile-group > div > div {
+  width: 33%;
+}
+
+.slug-selectable-tile-container .cds--tile-group .slug-radio-tile,
+.slug-selectable-tile-container .cds--tile-group .slug-selectable-tile {
+  margin-right: 1rem;
+  height: 200px;
+}
```

Comment:
Shouldn't we use logical properties in this part or it doesn't matter in this case?

Reply:
I updated the margin, but I don't think it matters too much for this example.